JavaScript/packages/recognizers-date-time/src/dateTime/constants.ts

```diff
@@ -73,6 +74,14 @@ export class Constants {
     static readonly Daytime: string = "TDT";
     static readonly Night: string = "TNI";
     static readonly BusinessHour: string = "TBH";
+
+    // Invalid year
+    public readonly InvalidYear: number = Number.MIN_VALUE;
```

Comment:
Did you use these constant somewhere?

Reply:
This part will soon be used to fix a bug. Shall I remove it now?

Reply:
Leave it then. Which PR will use it?

Reply:
The JS Datetime now can only recognize the date and will fail some test because of some missing function. I'm still working on it now.